Sprint-3/quote-generator/index.html

```diff
@@ -3,13 +3,17 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Title here</title>
+    <title>Quote generator app</title>
+    <link rel="stylesheet" href="style.css">
     <script defer src="quotes.js"></script>
   </head>
   <body>
-    <h1>hello there</h1>
-    <p id="quote"></p>
-    <p id="author"></p>
+    <div class="quote">
+      <p id="quote"></p>
```

Comment:
I can't open this on my machine , but from what i can see you dont have the big quotes at the start. That's a part of it too! Have a look at the sample image again